### PR TITLE
Create new OpenEBS version 3.3.0

### PR DESCRIPTION
**Pull request by emosbaugh**
#### What this PR does / why we need it:

Create new OpenEBS version 3.3.0
Workaround [chart issue](https://github.com/openebs/charts/issues/309) installing version 3.3.0.

#### Which issue(s) this PR fixes:

Fixes NONE

#### Does this PR introduce a user-facing change?

```release-note
Adds new [OpenEBS add-on](https://kurl.sh/docs/add-ons/openebs) version 3.3.0.
```

#### Does this PR require documentation?

NONE

### CODE REVIEW

File: addons/openebs/template/generate.sh [hunk 1]
```diff
@@ -23,6 +23,11 @@ function generate() {
         --set defaultStorageConfig.enabled=false \
         --set localprovisioner.enableDeviceClass=false \
         --set localprovisioner.enableHostpathClass=false \
+        --set release.version="$chart_version" \
```
Comment (emosbaugh) [inline addons/openebs/template/generate.sh:26]: this hardcodes imageTags from README.md
https://github.com/openebs/charts/blob/openebs-3.3.0/charts/openebs/README.md?plain=1#L90

**Comment by emosbaugh** [comment]
[sc-56120]

File: addons/openebs/template/generate.sh (hunk 1 above)
Comment (laverya) [reply addons/openebs/template/generate.sh:26]: do we want to hardcode images, though?
won't this prevent us from getting a new ndm image if one comes out in the future?

we're not regenerating old versions, so I don't see the problem with letting them float

**Review by rrpolanco — APPROVED** [review]

File: addons/openebs/template/generate.sh (hunk 1 above)
Comment (emosbaugh) [reply addons/openebs/template/generate.sh:26]: I am working around an upstream issue I created. They forgot to update the images for the new version of the chart.

https://github.com/openebs/charts/issues/309

**Review by laverya — APPROVED** [review]

**Pull request merged**